Source/Phone/Call/DtmfQueue.swift

```diff
@@ -49,11 +47,11 @@ class DtmfQueue {
         }
     }
     
-    func push(_ event: String, completionHandler: CompletionHandler?) {
+    func push(participantUrl: String, deviceUrl: URL, event: String, completionHandler: CompletionHandler?) {
```

Comment:
I think that's related to the registration lifecycle as well. You can see it's a hot spot to refactor structurally.